Octokit/Clients/ICodesOfConductClient.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Octokit.Internal;
+
+namespace Octokit
+{
+    /// <summary>
+    /// GitHub client for the Codes of Conduct API.
+    /// </summary>
+    /// <remarks>
+    /// See the <a href="https://developer.github.com/v3/codes_of_conduct/">API documentation</a> for more information.
+    /// </remarks>
+    public interface ICodesOfConductClient
```

Comment:
The other build error is the missing corresponding client for our reactive API. It's failing because it expects`Octokit.Reactive/IObservableGitHubClient.cs` to have a similar API, which internally just calls your original API. 

Have a look at `Octokit.Reactive/Clients/ObservableMiscellaneousClient.cs` for an example of how this works, and it looks like you've found the right bits to add that it should look familiar.